contracts/MinimalBalance.sol

```diff
@@ -0,0 +1,54 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity ^0.8.20;
+
+import {Currency, CurrencyLibrary} from "./types/Currency.sol";
+import {IMinimalBalance} from "./interfaces/IMinimalBalance.sol";
+
+/// An intentionally barebone balance mapping only supporting mint/burn/transfer
+contract MinimalBalance is IMinimalBalance {
```

Comment:
Do we want to call it `MinimalClaims` instead for clarity? i.e. i think claim better explains what the balances refer to

Reply:
Or even just `Claims`? Theyre not being compared to anything else anymore given this is now _our_ implementation.

Reply:
I like Claims